### ISSUE TITLE
Export Tagged PDFs for better accessibility

### ISSUE THREAD

**Issue by beckej13820** [pressbooks/pressbooks#1695]
### Feature Description

PDFs would be more useful for screenreaders if they came tagged with the markup from the HTML. Currently exported digital distribution PDFs aren't tagged, but Prince has the options available. 

I was able to use prince to be able to make a tagged PDF from the command line, so I believe that it has the capability, it just needs an extra command.

https://www.princexml.com/doc-prince/#pdf-profiles

From the prince documentation, adding an extra command will make the PDFs tagged. "To enable tagged PDF without using one of the profiles that already imply it, the command-line option --tagged-pdf can be used."

### Feature Use Case

Tagged PDFs are indexed for assistive technology, so that users can easily move throughout the structure of the document.

### Other Notes

When I created a document on my computer with Prince using a Pressbooks source, the only errors thrown doing a PDF Full Check in Adobe Acrobat Pro were from Chapter names and Page numbers at the bottom of the page. I don't know if it is possible to add a tag to that element as the PDF is being created. I've attached the document here.

[Disease-Prevention-and-Healthy-Lifestyles-tagged.pdf](https://github.com/pressbooks/pressbooks/files/3124178/Disease-Prevention-and-Healthy-Lifestyles-tagged.pdf)

**Comment by beckej13820** [comment]
Or, as I look around at this. Maybe using PDF standard PDF-UA-1 would be the best for digital distribution.  --pdf-profile="PDF/UA-1"

"Generally speaking, PDF/A profiles were created with the goal of long-term preservation of the documents (PDF for Archiving), while PDF/X profiles were created to address publisher's needs in the graphic arts industry (PDF for eXchange). The PDF/UA profile is a standard for producing accessible electronic documents (PDF for Universal Accessibility)."

**Comment by SteelWagstaff** [comment]
See also conversation in https://pressbooks.community/t/pdf-export-not-transferring-alt-text/935/4

**Comment by connerbw** [comment]
# What changed in Prince 12?

> With Prince 12, content inside the page margins, such as headers and footers, can no longer be displayed outside of the margin area.

http://help.docraptor.com/pdf-styling-and-troubleshooting/troublingshooting-issues/working-with-prince-12s-new-page-margin-layout

Fix proposals inside ☝️ 

# Prince 12 release notes:

https://www.princexml.com/releases/12/

**Comment by mcgratay** [comment]
Testing notes: Exports failing on all digital PDFs.

**Comment by mcgratay** [comment]
Resolved.
